mps_youtube/commands/play.py

```diff
@@ -92,7 +84,21 @@ def play(pre, choice, post=""):
                 streams.preload(g.model[chosen + 1], override=override)
 
         try:
-            play_range(songlist, shuffle, repeat, override)
+            if not config.PLAYER.get or not util.has_exefile(config.PLAYER.get):
+                g.message = "Player not configured! Enter %sset player <player_app> "\
+                    "%s to set a player" % (c.g, c.w)
+                return
+            else:
+                if config.PLAYER.get.endswith("mpv.exe"):
+                    mpv = Mpv()
+                    mpv.play_range(songlist, shuffle, repeat, override)
+                elif config.PLAYER.get.endswith("mplayer.exe"):
+                    mplayer = Mplayer()
+                    mplayer.play_range(songlist, shuffle, repeat, override)
+                else:
+                    g.message = "Player not supported! Enter %sset player <player_app> "\
```

Comment:
Maybe we could do both. Provide a specific class for vlc and a general fallback for any other player. We will look at the code more precisely in order to see all the methods etc. needed for the other players. Would you prefer that in a different file, a new subclass for all other players or a general method-methods in Player class? I believe the second option is the best.